client/my-sites/sharing/buttons/preview-action.jsx

```diff
@@ -1,52 +1,44 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	classNames = require( 'classnames' );
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import { omit, startsWith, endsWith } from 'lodash';
 
-module.exports = React.createClass( {
-	displayName: 'SharingButtonsPreviewAction',
+const SharingButtonsPreviewAction = ( props ) => {
```

Comment:
Can we enumerate the props in the parameters list? Like:

```js
const SharingButtonsPreviewAction = ( {
   active,
   position,
   icon,
   children
} )
```

I would say it's easier to read the component signature/definition  as you don't have to parse it's body just to see what props can/needs to be passed into it.